sys/shell/commands/sc_gnrc_icmpv6_echo.c

```diff
@@ -295,16 +297,15 @@ static void _pinger(_ping_data_t *data)
     ipv6 = pkt->data;
     /* if data->hoplimit is unset (i.e. 0) gnrc_ipv6 will select hop limit */
     ipv6->hl = data->hoplimit;
-    if (data->iface > KERNEL_PID_UNDEF) {
+    if (data->netif != NULL) {
         gnrc_netif_hdr_t *netif;
```

Comment:
`netif` is no longer used according to GCC

Reply:
Removed